docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -282,3 +282,30 @@ $JBOSS_HOME/bin/elytron-tool.sh filesystem-realm-encrypt -i ./standalone/configu
 This command creates a new filesystem realm by taking the existing filesystem realm from the specified input location, `./standalone/configuration/fs-realm-plain`, and encrypting its contents using the secret key with alias `key` from the specified credential store, `./mycredstore.cs`. The new filesystem realm is stored in the specified output location, `./standalone/configuration/fs-realm-enc`.
 
 A `.cli` script will also be generated at the root of the filesystem realm. The script contains WildFly CLI commands that can be used to configure a `secret-key-credential-store` resource and a `filesystem-realm` resource in the Elytron subsystem that makes use of the newly encrypted realm content.
+
+== Adding Integrity Checking for an Existing Filesystem Realm
+
+Integrity checking can also be added to an existing filesystem realm using the WildFly Elytron Tool. This takes an existing filesystem realm and creates a new filesystem realm with integrity checking support. The integrity checking uses a set of private-public key pairs, where the private key is only shared with certain known and trusted parties. As a result, the filesystem is secured and someone with access to the identity files, can no longer tamper with the data. 
```

Comment:
s/a set of private-public key pairs/a public and private key pair

Reply:
"where the private key is only shared with certain known and trusted parties. As a result, the filesystem is secured and someone with access to the identity files, can no longer tamper with the data"

I think that could be removed and reworded to something like:

The key pair will be used to create a new filesystem realm where each identity will be signed using the private key. This makes it possible to verify that identities in the realm haven't been tampered with.

Reply:
fixed!